pkg/operator/resources/operator/operator.go

```diff
@@ -528,6 +527,40 @@ func createCDIListCRD() *extv1beta1.CustomResourceDefinition {
 				ShortNames: []string{"cdi", "cdis"},
 			},
 
+			Validation: &extv1beta1.CustomResourceValidation{
+				OpenAPIV3Schema: &extv1beta1.JSONSchemaProps{
+					Properties: map[string]extv1beta1.JSONSchemaProps{
+						"apiVersion": {
+							Type: "string",
+						},
+						"kind": {
+							Type: "string",
+						},
+						"metadata": {
+							Type: "object",
+						},
+
+						"spec": {
+							Properties: map[string]extv1beta1.JSONSchemaProps{
+								"imageRegistry": {
+									Type: "string",
+								},
+								"imageTag": {
+									Type: "string",
+								},
+								"imagePullPolicy": {
+									Type: "string",
```

Comment:
Is it possible to limit to valid values for k8s image pull policy (Always, IfNotPresent, Never)?

Reply:
> Is it possible to limit to valid values for k8s image pull policy (Always, IfNotPresent, Never)?

added the Enum which limits image pull policy values ```Always, IfNotPresent, Never.```